app/models/manageiq/providers/vmware/infra_manager/provision/customization.rb

```diff
@@ -219,6 +221,16 @@ def load_customization_spec(custom_spec_name)
     end
   end
 
+  def encryption_key
+    @encryption_key ||= source.ext_management_system.with_provider_connection do |vim|
+      vim.getVimCustomizationSpecManager&.encryptionKey&.map(&:to_i)
+    end
+  end
+
+  def encrypt_passwords?
+    !!get_option(:sysprep_encrypt_passwords)
```

Comment:
Is there ever a time we _wouldn't_ encrypt the passwords?  I'm wondering if we need an option at all.